Thu Feb 11 19:56:35 CET 2016 Thread[main,5,main] Cleanup action starting
java.sql.SQLException: Database '/home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator/out/production/CelluarAutomaton/automata/derby/CellularAutomaton' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:270)
	at cellularautomaton.controller.SettingsController.createConnection(SettingsController.java:49)
	at cellularautomaton.controller.SettingsController.<clinit>(SettingsController.java:30)
	at cellularautomaton.controller.CellularAutomatonController.<init>(CellularAutomatonController.java:76)
	at cellularautomaton.app.Main.createAutomatonWindow(Main.java:29)
	at cellularautomaton.app.Main.main(Main.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
Caused by: ERROR XJ004: Database '/home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator/out/production/CelluarAutomaton/automata/derby/CellularAutomaton' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 26 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database '/home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator/out/production/CelluarAutomaton/automata/derby/CellularAutomaton' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:270)
	at cellularautomaton.controller.SettingsController.createConnection(SettingsController.java:49)
	at cellularautomaton.controller.SettingsController.<clinit>(SettingsController.java:30)
	at cellularautomaton.controller.CellularAutomatonController.<init>(CellularAutomatonController.java:76)
	at cellularautomaton.app.Main.createAutomatonWindow(Main.java:29)
	at cellularautomaton.app.Main.main(Main.java:23)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
============= end nested exception, level (1) ===========
Cleanup action completed
----------------------------------------------------------------
Thu Feb 11 19:56:35 CET 2016:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.12.1.1 - (1704137): instance a816c00e-0152-d1b1-2b2d-000008cc2258 
on database directory /home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator/out/production/CelluarAutomaton/automata/derby/CellularAutomaton with class loader sun.misc.Launcher$AppClassLoader@75b84c92 
Loaded from file:/home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator/res/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_71-b15
user.dir=/home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator
os.name=Linux
os.arch=amd64
os.version=4.2.8-200.fc22.x86_64
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Feb 11 19:56:43 CET 2016:
Shutting down instance a816c00e-0152-d1b1-2b2d-000008cc2258 on database directory /home/vspadi/Dokumente/GitHub/ViktorSpadiSimulator/out/production/CelluarAutomaton/automata/derby/CellularAutomaton with class loader sun.misc.Launcher$AppClassLoader@75b84c92 
